### PR TITLE
Use latest gooddata-rest-common

### PR DESCRIPTION
**Pull request by jakub-plichta**
Close: #556

### CODE REVIEW

File: src/main/java/com/gooddata/GoodData.java [hunk 1]
```diff
@@ -58,7 +59,11 @@
  */
 public class GoodData {
 
-    public static final String GDC_REQUEST_ID_HEADER = "X-GDC-REQUEST";
+    /**
+     * @deprecated use {@link Header#GDC_REQUEST_ID} instead.
+     */
+    @Deprecated
+    public static final String GDC_REQUEST_ID_HEADER = Header.GDC_REQUEST_ID;
```
Comment (jimirocks) [inline src/main/java/com/gooddata/GoodData.java]: does this belong rather to the second commit?

Comment (jakub-plichta) [reply src/main/java/com/gooddata/GoodData.java]: actually not - second commit is for upgrade to 1.1.0 version.. I'll reorganize commits so this fact is more visible

Comment (liry) [reply src/main/java/com/gooddata/GoodData.java]: Plz, also change all usages of this constant (GDC_REQUEST_ID_HEADER) in this codebase to new one (Header.GDC_REQUEST_ID) not to use deprecated one.

Comment (jakub-plichta) [reply src/main/java/com/gooddata/GoodData.java]: done and done

**Pull request merged**